tools/webhook_test.py

```diff
@@ -171,23 +166,32 @@ def get_and_validate_team():
 print "What kind of webhook would you like to send?(put in a number)\n"+webhooks_formatted+">",
 type = whtypes.get(raw_input(), 0)
 if type == 0:
-    log.error("Must put in valid webhook type")
+    print "Must put in valid webhook type"
     sys.exit(1)
 
 payload = set_init(type)
 
-print "What is the URL of where you would like to send the webhook? (default: 127.0.0.1:4000)\n>",
+print "What is the URL of where you would like to send the webhook? (default: http://127.0.0.1:4000)\n>",
 url = raw_input()
 if url == '' or url.isspace():
     url = "http://127.0.0.1:4000"
-    log.info("Assuming " + url + " as webhook URL")
+    print "Assuming " + url + " as webhook URL"
 
 print "Does location matter?\n>",
```

Comment:
We should consider also asking in this question if a user uses geofences.  Because if they do, then location DOES matter whether they think it does or not.